### PR TITLE
12 - Create zarf package

### PR DESCRIPTION
**Pull request by CollectiveUnicorn**
* Creates a Zarf package for leapfrog-ui
* Moves from `configs.json` to environment variables and a `.env` file
* Updates docker file to create a production build
* Sets version to 0.0.1 and replaces hard coded values with Zarf variables
* Removes folders

### CODE REVIEW

File: manifests/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    metadata:
+      labels:
+        app: leapfrog-ui
+    spec:
+      containers:
+        - name: leapfrog-ui
+          image: "ghcr.io/defenseunicorns/leapfrogai/leapfrog-ui:###ZARF_VAR_IMAGE_VERSION###"
+          ports:
+            - containerPort: 3000
+          env:
```
Comment (justinthelaw) [inline manifests/deploy.yaml:21]: Not strictly necessary, but may be a good idea to add defaults for some of these variables based on the .env file?

**Review by justinthelaw — APPROVED** [review]
Looks and works great - had a minor comment on the zarf.yaml, but otherwise everything else is good to me!

File: manifests/deploy.yaml (hunk 1 above)
Comment (CollectiveUnicorn) [reply manifests/deploy.yaml:21]: Do you mean just the container port or the Zarf variables in general? They should have default values already that are largely based on the `.env` file. Most of these should "just work" aside from some of URI things that will differ by environment.

Comment (CollectiveUnicorn) [reply manifests/deploy.yaml:21]: Gonna close this out for now, if you can open a separate issue for this I'll create a separate PR to address it.

**Pull request merged**